### ISSUE TITLE
Feature Request: Pin rows (i.e. exclude from sorting)

### ISSUE THREAD

**Issue by s0x** [swimlane/ngx-datatable#383]
**I'm submitting a ...**
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here
```

**Expected behavior**
Allow to pin rows which would exclude them from sorting. This could be either a property of the row or a table attribute pinning the first/last n rows (e.g. [frozenTop]="1"). I guess the former option would be hard to handle if someone tries to pin rows in the middle of the table.

**What is the motivation / use case for changing the behavior?**
I wanna realize an file-browser element. The first row should be a "go to parent"-element (containing the label ".."). This row should be excluded from sorting and stick at the top of the list. A similar use-case would be to have a summary-row at the bottom of the table which should stick at the bottom even if sorting has been triggered

**Comment by sachinofficial** [comment]
I agree 👍 much needed feature, the way that I am currently working around this is by checking the column `prop` in the `onSort($event)` callback and doing nothing in-case of such columns.

**Comment by amcdnl** [comment]
Thats not really true 'pinning' since when you scroll its gone. Its a pretty hefty effort to accomplish this but if you're interested i can point you in the direction.

**Comment by racpa** [comment]
How would go about this?
